### ISSUE TITLE
RuntimeError: you have an eventloop running but not using Jupyter/ipython.

### ISSUE THREAD

**Issue by ColdCawfee** [jina-ai/dalle-flow#92]
I get this when attempting to use VSC (Visual Studio Code) to run DocArray's Document class. It keeps saying to adjust "Client/Flow" to "asyncio=True", but I am not using either of those classes.

Any help would be awesome, thank you!

**Comment by ColdCawfee** [comment]
Logs attached if that helps too:
![Screenshot 2022-08-27 004403](https://user-images.githubusercontent.com/79474598/187013139-9fe5e70d-de01-48b1-893e-4f56b26a2f3b.png)

**Comment by AmericanPresidentJimmyCarter** [comment]
See this issue: https://github.com/jina-ai/jina/issues/4761

**Comment by AmericanPresidentJimmyCarter** [comment]
Note that if you're trying to use this directly in async code you should alter `flow.tmp.yml` to have

```yaml
jtype: Flow
with:
  asyncio: True
```

It should perhaps be another environmental variable in `flow_parser.py` to make the feature easily accessible. @delgermurun 

Yet the default configuration is sync -- hence to work with async code as specified I actually use a [filesystem bus in my Discord bot code](https://github.com/AmericanPresidentJimmyCarter/yasd-discord-bot/blob/master/imagetool.py), just for the sake of making it easy on people who are pulling or creating the docker image directly.

**Comment by delgermurun** [comment]
@AmericanPresidentJimmyCarter yeah, let's have env variable for it. Would you mind raising a PR 😁 ?

**Comment by AmericanPresidentJimmyCarter** [comment]
No problem, I will look into it this week.
